Runner/script.sh.template

```diff
@@ -17,17 +19,17 @@ mkdir ../artifacts-pr
 mkdir ../clr-checked-main
 mkdir ../clr-checked-pr
 
-git checkout main
+git switch main
 ./build.sh clr+libs -c Release
-./build.sh clr -c Checked
+./build.sh clr.jit -c Checked
 
 cp -r artifacts/bin/coreclr/linux.x64.Release/* ../artifacts-main
 cp -r artifacts/bin/runtime/net8.0-linux-Release-x64/* ../artifacts-main
 cp -r artifacts/bin/coreclr/linux.x64.Checked/* ../clr-checked-main
 
-git checkout pr
-./build.sh clr.corelib+clr.nativecorelib+libs.pretest -c Release
-./build.sh clr -c Checked
+git switch pr
+./build.sh clr+libs.pretest -c Release
```

Comment:
```suggestion
./build.sh clr+libs.pretest+libs -c Release
```

Let's hope the build infra is smart enough to skip most work if there are no changes 🤷‍♂️